src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -72,30 +57,31 @@ export default class AirportModel {
         this.radio = null;
         this.level = null;
         this._positionModel = null;
-        this.runways = [];
+        this._runwayCollection = null;
+        // TODO: rename to `runwayName`
+        //
+        this.runway = null;
```

Comment:
looks to be used as a default runway of some sort?  I can't tell if its actually getting used, though.